test/jdk/java/foreign/StdLibTest.java

```diff
@@ -53,6 +60,8 @@ public class StdLibTest extends NativeTestHelper {
 
     final static Linker abi = Linker.nativeLinker();
 
+    final static Charset charSetNative = Charset.forName(System.getProperty("native.encoding"));
```

Comment:
Normally modifiers should appear in canonical order (`static final`), but it's reasonable to use the same order as line 61.

The name feels awkward to me - `S` should not be capitalized (it's not in "Charset"), and "native" is an adjective here which normally precedes the thing it modifies: how about `nativeCharset` or `nativeEncoding`?

Reply:
That sounds more clear - changed in https://github.com/ibmruntimes/openj9-openjdk-jdk/compare/3c83057c855478512324c41571b616ee813d3234..6dcafe07f14b47747bef44a718ed060073f7d53e